docs/src/Callout.doc.js

```diff
@@ -0,0 +1,100 @@
+// @flow strict
+import * as React from 'react';
+import PropTable from './components/PropTable.js';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="Callout"
+  />
+);
+
+card(
+  <PropTable
+    props={[
+      {
+        name: 'description',
+        type: 'string',
+        required: true,
+      },
+      {
+        name: 'iconAccessibilityLabel',
+        type: 'string',
+        required: true,
+      },
+      {
+        name: 'onDismiss',
+        type: '() => void',
+        required: true,
+      },
+      {
+        name: 'primaryLink',
+        type: '{| href: string, label: string |}',
+      },
+      {
+        name: 'secondaryLink',
+        type: '{| href: string, label: string |}',
+      },
+      {
+        name: 'style',
+        type: `"error" | "info" | "warning"`,
+        required: true,
+      },
+      {
+        name: 'title',
+        type: 'string',
+      },
+    ]}
+  />
+);
+
+card(
+  <Example
+    name="Info Example"
+    defaultCode={`
+<Callout
+    style="info" 
```

Comment:
@norrise120 could you enforce 2 spaces indentation? (just like you did in the example below)